handlers/bounty_test.go

```diff
@@ -1197,22 +1200,70 @@ func TestMakeBountyPayment(t *testing.T) {
 
 		return mockClient, nil
 	}
-	bHandler := NewBountyHandler(mockHttpClient, mockDb)
-
-	unauthorizedCtx := context.WithValue(ctx, auth.ContextKey, "")
-	authorizedCtx := context.WithValue(ctx, auth.ContextKey, "valid-key")
+	bHandler := NewBountyHandler(mockHttpClient, db.TestDB)
 
 	var mutex sync.Mutex
 	var processingTimes []time.Time
 
-	bountyID := uint(1)
+	now := time.Now().Unix()
+	bountyOwnerId := "owner_pubkey"
+
+	db.TestDB.DeleteWorkspaceBudget()
```

Comment:
@AbdulWahab3181 why do you have to delete the Workspace Budget, the budgets are different for different workspaces

Reply:
@elraphty I deleted for safe side. Should I remove?

Reply:
Yeah remove it

Reply:
@elraphty Addressed